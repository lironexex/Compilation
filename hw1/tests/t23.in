//strings 3 - empty string
"a string contains *zero* or more characters, for example, this is legal:"
"" // a zero-character string
