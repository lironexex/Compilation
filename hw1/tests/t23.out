1 COMMENT 1
2 STRING a string contains *zero* or more characters, for example, this is legal:
3 STRING 
3 COMMENT 1
